<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddClass</class>
 <widget class="QWidget" name="AddClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background_image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="classname_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF; /* White text color */
    background-color: #004080; /* Dark blue background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 8px 12px; /* Padding */
    border-radius: 5px; /* Rounded corners */
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="classcode_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF; /* White text color */
    background-color: #004080; /* Dark blue background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 8px 12px; /* Padding */
    border-radius: 5px; /* Rounded corners */
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;Code&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="classlecturehall_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF; /* White text color */
    background-color: #004080; /* Dark blue background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 8px 12px; /* Padding */
    border-radius: 5px; /* Rounded corners */
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;Lecture Hall&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="classtime_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF; /* White text color */
    background-color: #004080; /* Dark blue background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 8px 12px; /* Padding */
    border-radius: 5px; /* Rounded corners */
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;Time&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="class_title">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF; /* White text color */
    background-color: #004080; /* Dark blue background */
    font-size: 10px; /* Font size */
    font-weight: bold; /* Bold font weight */
    padding: 8px 12px; /* Padding */
    border-radius: 5px; /* Rounded corners */
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;Class Info&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="classname_text">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>135</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* White background */
    border: 1px solid #004080; /* Dark blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px; /* Adjust padding as needed */
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="classcode_text">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* White background */
    border: 1px solid #004080; /* Dark blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px; /* Adjust padding as needed */
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="classlecturehall_text">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF; /* White background */
    border: 1px solid #004080; /* Dark blue border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px; /* Adjust padding as needed */
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="class_submit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>330</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #0077b3; /* Medium blue background for buttons */
    color: #FFFFFF; /* White text */
    border: none; /* No border */
    border-radius: 5px; /* Rounded corners */
    padding: 8px 16px; /* Adjust padding as needed */
}

QPushButton:hover {
    background-color: #4da6ff; /* Lighter blue on hover */
}

QPushButton:pressed {
    background-color: #005c99; /* Darker blue when pressed */
}
</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

background-color: rgb(10, 104, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #0077b3; /* Medium blue background for buttons */
    color: #FFFFFF; /* White text */
    border: none; /* No border */
    border-radius: 5px; /* Rounded corners */
    padding: 8px 16px; /* Adjust padding as needed */
}

QPushButton:hover {
    background-color: #4da6ff; /* Lighter blue on hover */
}

QPushButton:pressed {
    background-color: #005c99; /* Darker blue when pressed */
}
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
